src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -280,7 +280,9 @@
                 "hostname": "asw.dc",
                 "bgp_asn": "64850",
                 "hwsku": "Stone",
-                "buffer_model": "dynamic"
+                "buffer_model": "dynamic",
+                "cloudtype": "None",
```

Comment:
>cloudtype [](http://example.com/codeflow?start=17&length=9)

We have some sample strings, could you use them? [https://github.com/Azure/sonic-buildimage/blob/b17e92b309f593ba629eb163e48a692f34df49dc/src/sonic-config-engine/tests/simple-sample-graph-metadata.xml#L218-L227](https://github.com/Azure/sonic-buildimage/blob/b17e92b309f593ba629eb163e48a692f34df49dc/src/sonic-config-engine/tests/simple-sample-graph-metadata.xml#L218-L227)